docs/en_US/Compressor/Pruner.md

```diff
@@ -398,5 +402,176 @@ We try to reproduce the experiment result of the fully connected network on MNIS
 The above figure shows the result of the fully connected network. `round0-sparsity-0.0` is the performance without pruning. Consistent with the paper, pruning around 80% also obtain similar performance compared to non-pruning, and converges a little faster. If pruning too much, e.g., larger than 94%, the accuracy becomes lower and convergence becomes a little slower. A little different from the paper, the trend of the data in the paper is relatively more clear.
 
 
+## NetAdapt Pruner
```

Comment:
The order of each section is better consistent with the content directory at beginning.

Reply:
fixed